.mci.yml

```diff
@@ -78,7 +78,7 @@ tasks:
           params:
               working_dir: "mongo-cxx-driver/build"
               script: |
-                  PKG_CONFIG_PATH="/data/tmp/c-driver-install/lib/pkgconfig" /opt/cmake/bin/cmake -DCMAKE_BUILD_TYPE=${build_type} -DCMAKE_INSTALL_PREFIX=./install ..
+                  PKG_CONFIG_PATH="/data/tmp/c-driver-install/lib/pkgconfig" /opt/cmake/bin/cmake -DCMAKE_BUILD_TYPE=${build_type} -DCMAKE_INSTALL_PREFIX=./install -DCMAKE_C_FLAGS="-Wall -Wextra -Wno-attributes -Werror" -DCMAKE_CXX_FLAGS="-Wall -Wextra -Wno-attributes -Werror" ..
```

Comment:
Will these flags change depending on the buildvariant?  If not then it's fine to keep them here.  Otherwise we might want to pull these into a variable, then attach that to each buildvariant and use mci's expansions:
```
cmake_flags:  &standard_warning_flags '-DCMAKE_C_FLAGS="-Wall -Wextra -Wno-attributes -Werror" -DCMAKE_CXX_FLAGS="-Wall -Wextra -Wno-attributes -Werror"'

// in the script:
PKG_CONFIG_PATH="/data/tmp/c-driver-install/lib/pkgconfig" /opt/cmake/bin/cmake -DCMAKE_BUILD_TYPE=${build_type} -DCMAKE_INSTALL_PREFIX=./install ${cmake_flags} ..

// buildvariant
- name: ubuntu1404
  expansions:
     cmake_flags: *standard_warning_flags
```

But I think this is overkill unless these flags are different on different systems.  We can always do it later if we find we need it.